src/constants.ts

```diff
@@ -7,6 +7,9 @@ export const jsx: ParserPlugin = 'jsx';
 
 export const newLineCharacters = '\n\n';
 
+export const ChunkSideEffectNode = 'side-effect-node';
```

Comment:
I think the variable name should start lowercase to follow the project consistency. 
```suggestion
export const chunkSideEffectNode = 'side-effect-node';
```

Reply:
Ah yeah you're right, I usually try to follow existing conventions, but I missed that.